tests/functional/n/no/no_member_imports.py

```diff
@@ -33,9 +33,9 @@ def test_ignored_modules_root_one_applies_as_well() -> None:
 
 
 def test_ignored_modules_patterns() -> None:
-    import collections
+    import importlib
 
-    collections.abc.THIS_does_not_EXIST
+    importlib.metadata.THIS_does_not_EXIST
```

Comment:
collections.abc is now an attribute. i don't love it, but python forced our hand. collections is not essential to this test. someone might file an issue eventually.